package.json

```diff
@@ -69,7 +69,8 @@
     "uiFramework:start": "cd packages/kbn-ui-framework && yarn docSiteStart",
     "uiFramework:build": "cd packages/kbn-ui-framework && yarn docSiteBuild",
     "uiFramework:createComponent": "cd packages/kbn-ui-framework && yarn createComponent",
-    "uiFramework:documentComponent": "cd packages/kbn-ui-framework && yarn documentComponent"
+    "uiFramework:documentComponent": "cd packages/kbn-ui-framework && yarn documentComponent",
+    "kbn:watch": "node scripts/kibana --dev"
```

Comment:
Is there some option we can give Kibana so it outputs the "normal logs" instead of the json formatting?

Reply:
Good catch, yeah we can use `--logging.json=false`. Surprisingly it's `true` when Kibana process is run by `kbn watch` (it's not in TTY mode, so https://github.com/elastic/kibana/blob/master/src/server/config/schema.js#L117 turns on json logging).